wandb/sdk/internal/sender_config.py

```diff
@@ -72,9 +72,9 @@ def _add_unset_keys_from_subtree(
         new_subtree = _subtree(self._tree, path, create=True)
         assert new_subtree is not None
 
-        for chart_key, chart_value in old_subtree.items():
-            if chart_key not in new_subtree:
-                new_subtree[chart_key] = chart_value
+        for key, value in old_subtree.items():
+            if key not in new_subtree:
+                new_subtree[key] = value
```

Comment:
interesting that we don't cover this... (no need to add tests, this code is going away soon)

Reply:
ah, yeah i was hoping this would be covered by the resume tests, but i suppose not..